docs/spec_draft.md

```diff
@@ -941,6 +942,69 @@ produces an implementation-defined value.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.dynamic_update_slice
+
+### Semantics
+
+Overwrites the `operand` tensor at `start_indices` with values from `update`
+tensor and produces a `result` tensor. More formally,
+`result[start_indices0+j0, ..., start_indicesR-1+jR-1] = update[j0, ..., jR-1]`
+and `result[i0, ..., iR-1] = operand[i0, ..., iR-1]` otherwise where `R` $=$
+rank(`operand`).
+
+### Inputs
+
+| Name            | Type                                                     |
+|-----------------|----------------------------------------------------------|
+| `operand`       | tensor of any supported type                             |
+| `update`        | tensor of any supported type                             |
+| `start_indices` | variadic number of 0-dimensional tensors of integer type |
```

Comment:
Same comment as in #556. Both HLO and MHLO agree on indices being tensors of integer type. Any particular reason to restrict that to si64?

Also, regardless of the decision here, we should make sure that all indices have the same type. (This will be automatic if we decide to stick to `si64`).

Reply:
An oversight. Please ignore.